internal/tiltfile/k8s.go

```diff
@@ -871,6 +875,9 @@ type portForward struct {
 var _ starlark.Value = portForward{}
 
 func (f portForward) String() string {
+	if f.Name != "" {
```

Comment:
it seems really weird to invent a new syntax here, how about:

fmt.Sprintf("port_forward(local_port=%d, container_port=%d, name=%q)", f.LocalPort, f.ContainerPort, f.Name)

Reply:
fairrrr!